### PR TITLE
Dockerize

### LINKED ISSUE #1: Dockerize
期待

### PR DESCRIPTION
**Pull request by morishin**
closes #1 

どんどんドッカーどーんと行こう

Please review. 😉 @ryota-ka

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -66,6 +66,24 @@ Option | Description | Example values
...
+### 2. Run
+If you put `tweets.csv` in project root directory, for example, you can run with the following command.
+```shell
+$ cd markov-bot
+$ docker run -d -e "MARKOV_BOT_CONSUMER_KEY=<YOUR_CONSUMER_KEY>" \
+-e "MARKOV_BOT_CONSUMER_SECRET=<YOUR_CONSUMER_SECRET>" \
+-e "MARKOV_BOT_ACCESS_TOKEN=<YOUR_ACCESS_TOKEN>" \
+-e "MARKOV_BOT_ACCESS_TOKEN_SECRET=<YOUR_TOKEN_SECRET>" \
+-v $(pwd)/tweets.csv:/markov-bot/tweets.csv \
+markov-bot -- --tweets-csv ./tweets.csv --interval 1800
```
Comment (morishin) [inline README.md:86]: It does not work without `--`. (I spent a lot of time finding this...)

I tried using `--` in `ENTRYPOINT` but it does not work too.

``` dockerfile
# not work
ENTRYPOINT ["stack", "exec", "markov-bot-exe", "--"]
docker run ... markov-bot --tweets-csv ./tweets.csv --interval 1800
```

**Comment by morishin** [comment]
@kohey18 Docker Ojisaaaaan, I'd like to ask you to review this too.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,17 @@
+FROM samdoshi/haskell-stack:latest
+
+RUN apt-get update
+RUN apt-get install -y mecab libmecab-dev mecab-ipadic-utf8
```
Comment (ryota-ka) [inline Dockerfile]: These two commands could be concatenated with `&&`, and should be so, I suppose.

Comment (morishin) [inline Dockerfile]: :v: ca49afe

**Comment by ryota-ka** [comment]
please squash your commits before merging! thanks!!

[LGTM](http://lgtm.camph.net/i/8znP3rEDN)

File: README.md (hunk 1 above)
Comment (ryota-ka) [reply README.md:86]: `["stack", "exec", "--", "markov-bot-exe"]` will do.

**Pull request merged**